Parsers/ASimNetworkSession/Tests/SentinelOne_ASimNetworkSession_DataTest.csv

```diff
@@ -0,0 +1,7 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 11382 records (100.0%) for field [EventProduct] of type [Enumerated]: [""SentinelOne""] (Schema:NetworkSession)"
```

Comment:
pls add SentinelOne as supported value for EventProduct and EventVendor in Tester.csv